django/cantusdb_project/main_app/management/commands/update_cached_concordances.py

```diff
@@ -10,7 +10,7 @@
 
 class Command(BaseCommand):
     def handle(self, *args, **kwargs) -> None:
-        CACHE_DIR: str = "api_cache"
+        CACHE_DIR: str = "/resources/api_cache"
```

Comment:
You should probably configure this by command-line parameter. Running this locally on my machine will put this in the root of my system, which is not good.

Reply:
If this is saving to your root directory, I'm guessing you're not running the project in Docker?

In my local setup, this saves to `/resources/api_cache` within the django container, right where it needs to be to get picked up by the `api_cache` volume specified in our `docker-compose.yml` file.

Unless I'm missing something - considering this command has been designed to save the concordances to a Docker volume, and all our documentation makes it clear that the application is meant to be run in Docker, I feel like configuring this via the command line would just add complexity.

Would adding a comment here - something along the lines of "this is the path for the `api_cache` volume configured in `docker-compose.yml`" - help?

Reply:
Adding an output directory configuration is pretty standard. You can also set a default value if you want. But the option to change the location, should it be useful, is really handy.

(I don’t run docker. I’ve deployed a number of Django sites to production, and they didn’t need Docker either. I actually think this application doesn’t need docker, and that you would get better performance and have a much simpler deploy process if you got rid of it. 

But it works for now, and there are other more important things to do. )

Reply:
I wasn't fully absorbing what you meant by "by command-line parameter"... Just pushed a commit that allows you to run `python manage.py update_cached_concordances -d (or --directory) "/a/filepath/of/my/choosing"`. Thanks for this suggestion!